services/worker-manager/schemas/v1/worker-lifecycle.yml

```diff
@@ -30,5 +30,18 @@ properties:
       will be terminated.  This value also dictates the lifetime of the
       temporary credentials granted to the worker, meaning that it must be
       less than 30 days. The default value is 4 days.
+  claimTimeout:
```

Comment:
Claim timeout sounds like something different - it sounds like how long the claim is, until reclaimTask needs to be called. Normally that is 20 minutes.

Perhaps notClaimingTimeout / betweenClaimsTimeout / offlineTimeout / awolTimeout / queueSilenceTimeout ?

Reply:
thanks for the feedback! I'm open to changes here to get more clarity.

`queueInactivityTimeout` ?

Reply:
renamed it